cs/roadmap.html

```diff
@@ -0,0 +1,242 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+    <title>Vývoj</title>
+    <link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet" type="text/css" />
+    <!-- <link rel="stylesheet" href="/static/css/base.css" type="text/css" media="screen" /> -->
+    <!-- <link rel="stylesheet" id="current-theme" href="/static/css/style.css" type="text/css" media="screen" /> -->
+    <link rel="stylesheet" href="/static/external/css/reset.css" type="text/css" media="screen" />
+    <link rel="stylesheet" href="/static/external/css/responsive-gs-12col.css" type="text/css" media="screen" />
+    <link rel="stylesheet" href="/static/css/base.css" type="text/css" media="screen" />
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:300,400,700|Gentium+Basic" rel="stylesheet" type="text/css" />
+    <link rel="icon" href="/static/SymPy-Favicon.ico" />
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
+    <script type="text/javascript" src="/static/javascript/main.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1><a href="index.html"> <img src="/static/images/logo.png" alt="SymPy Logo" /> SymPy</a></h1>
+        <div id="mobile-menu"><i class="icon-ellipsis-vertical"></i></div>
+      </header>
+      <nav id="main-navigation" class="row">
+        
+        
+        <ul>
+          <li class=""><a href="index.html">Hlavní stránka</a></li>
+          <li class=""><a href="features.html">Vybavení</a></li>
+          <li class=""><a href="download.html">Stáhnout</a></li>
+          <li class=""><a href="http://docs.sympy.org">Dokumentace <i class="icon-external-link-sign"></i></a></li>
+          <li class=""><a href="support.html">Podpora</a></li>
+          <li class=""><a href="development.html">Vývoj</a></li>
+          <li class="active"><a href="roadmap.html">Roadmap</a></li>
+          <li class=""><a href="donate.html">Donate</a></li>
+          <li><a href="http://live.sympy.org/">Online Konzole <i class="icon-external-link-sign"></i></a></li>
+        </ul>
+        
+      </nav>
+      <section class="row" id="main">
+        <div class="col span_7 clr small_span_4">
+          
+          <article id="content">
+            
+
+<h1 class="title">Roadmap</h2>
+<div class="inner">
+  <p class="first">
+    Broadly speaking, any form of poorly symbolic mathematical computation is
```

Comment:
It means I misspelt "purely" and wasn't paying attention when my spellchecker fixed it.